client/console/src/main/java/org/apache/syncope/client/console/panels/search/AnyObjectSearchPanel.java

```diff
@@ -88,6 +91,18 @@ protected void populate() {
                         null).stream().collect(Collectors.toMap(GroupTO::getKey, GroupTO::getName));
             }
         };
+
+        this.anames = new LoadableDetachableModel<Map<String, PlainSchemaTO>>() {
+
+            private static final long serialVersionUID = 5275935387613157437L;
+
+            @Override
+            protected Map<String, PlainSchemaTO> load() {
+                return schemaRestClient.<PlainSchemaTO>getSchemas(
```

Comment:
You shouldn't get available attribute names to query from the mapping, rather from underlying connector's schema.